### PR TITLE
sip2: improve command line utilities

### PR DESCRIPTION
**Pull request by lauren-d**
* Adds some CLI command to manage selfcheck terminals.
* Improves error messages in SIP2 handlers.
* Updates invenio-sip2 package.

Co-Authored-by: Lauren-D <laurent.dubois@itld-solutions.be>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

* rero/rero-ils-ui#<xx>

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

File: rero_ils/modules/selfcheck/cli.py [hunk 1]
```diff
@@ -42,24 +46,92 @@
...
+
         click.secho(f'login: {name}', fg='blue')
-        click.secho(f'token: {token.access_token}', fg='blue')
+        click.secho(f'token: {access_token}', fg='blue')
+
+
+@click.command('list_terminal')
+@with_appcontext
+def list_terminal():
+    """List all configured terminal."""
```
Comment (iGormilhit) [inline rero_ils/modules/selfcheck/cli.py:93]:
```suggestion
    """List all configured terminals."""
```

**Review by iGormilhit — COMMENTED** [review]
Commit message approved.

File: rero_ils/modules/selfcheck/cli.py (hunk 1 above)
Comment (lauren-d) [reply rero_ils/modules/selfcheck/cli.py]: ok. Done.

**Review by rerowep — CHANGES_REQUESTED** [review]
If I give a non existing user I get:
```
  File "//Users/peter/devel/virtualenvs/rero-ils-CoBhSWGK-py3.7/lib/python3.7/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'NoneType' object has no attribute 'id'
```
A message: User does not exist, would be better.

I'am not sure it is a valid use case but if I try to use the same user for a second terminal I get following warning:
```
WARNING (psycopg2.errors.UniqueViolation) duplicate key value violates unique constraint "ix_oauth2server_token_access_token"
DETAIL:  Key (access_token)=(\x58724f593842594f6e476c755242316579364a766e6f5350724b3469576e67695a7756467a7469484c57544e75784a677a4f724d34492f38785a67453568564d495a6d754d7345446e44316d6d69414b335a704a45413d3d) already exists.

[SQL: INSERT INTO oauth2server_token (client_id, user_id, token_type, access_token, refresh_token, expires, _scopes, is_personal, is_internal) VALUES (%(client_id)s, %(user_id)s, %(token_type)s, %(access_token)s, %(refresh_token)s, %(expires)s, %(_scopes)s, %(is_personal)s, %(is_internal)s) RETURNING oauth2server_token.id]
[parameters: {'client_id': '41ofFMveiwFuMxGadBPzIwMFlMSeZhszM7LkY8rj', 'user_id': 5, 'token_type': 'bearer', 'access_token': <psycopg2.extensions.Binary object at 0x106f5d270>, 'refresh_token': None, 'expires': None, '_scopes': '', 'is_personal': True, 'is_internal': False}]
(Background on this error at: http://sqlalche.me/e/13/gkpj)
```

**Review by iGormilhit — APPROVED** [review]

**Comment by lauren-d** [comment]
> If I give a non existing user I get:
> 
> ```
>   File "//Users/peter/devel/virtualenvs/rero-ils-CoBhSWGK-py3.7/lib/python3.7/site-packages/werkzeug/local.py", line 347, in __getattr__
>     return getattr(self._get_current_object(), name)
> AttributeError: 'NoneType' object has no attribute 'id'
> ```
> 
> A message: User does not exist, would be better.

Done.

> I'am not sure it is a valid use case but if I try to use the same user for a second terminal I get following warning:
> 
> ```
> WARNING (psycopg2.errors.UniqueViolation) duplicate key value violates unique constraint "ix_oauth2server_token_access_token"
> DETAIL:  Key (access_token)=(\x58724f593842594f6e476c755242316579364a766e6f5350724b3469576e67695a7756467a7469484c57544e75784a677a4f724d34492f38785a67453568564d495a6d754d7345446e44316d6d69414b335a704a45413d3d) already exists.
> 
> [SQL: INSERT INTO oauth2server_token (client_id, user_id, token_type, access_token, refresh_token, expires, _scopes, is_personal, is_internal) VALUES (%(client_id)s, %(user_id)s, %(token_type)s, %(access_token)s, %(refresh_token)s, %(expires)s, %(_scopes)s, %(is_personal)s, %(is_internal)s) RETURNING oauth2server_token.id]
> [parameters: {'client_id': '41ofFMveiwFuMxGadBPzIwMFlMSeZhszM7LkY8rj', 'user_id': 5, 'token_type': 'bearer', 'access_token': <psycopg2.extensions.Binary object at 0x106f5d270>, 'refresh_token': None, 'expires': None, '_scopes': '', 'is_personal': True, 'is_internal': False}]
> (Background on this error at: http://sqlalche.me/e/13/gkpj)
> ```

it's corrected

**Review by benerken — APPROVED** [review]

**Review by rerowep — APPROVED** [review]

**Pull request merged**